packages/webapp/src/pages/members/[id].tsx

```diff
@@ -1,27 +1,40 @@
 import { GetServerSideProps } from "next";
 
-import { RawLayout } from "_app";
-import { getMember, MemberCard, MemberCollections, MemberData } from "members";
+import { CallToAction, Card, RawLayout, SingleColLayout } from "_app";
+import {
+    getMember,
+    MemberCard,
+    MemberCollections,
+    MemberData,
+    MemberHoloCard,
+} from "members";
 
 interface Props {
     member?: MemberData;
 }
 
 export const MemberPage = ({ member }: Props) => {
     return member ? (
-        <RawLayout title={`${member.name}'s Profile`}>
-            <MemberCard member={member} />
+        <RawLayout>
+            <Card>
+                <div className="flex justify-center items-center space-y-10 xl:space-y-0 xl:space-x-10 flex-col xl:flex-row">
+                    <div className="max-w-xl">
+                        <MemberHoloCard member={member} />
+                    </div>
+                    <MemberCard member={member} />
+                </div>
+            </Card>
```

Comment:
What if this entire <Card> is the <MemberCard>? I'm trying to think fo a case where we wnat the <MemberCard> without the HoloCard

Reply:
I’m going to be doing some more work on this page today. I’ll def take a look at consolidating this. Thanks!